brave-intl/publishers
# Release 2.25.22 - Eyeshade Balances and Client, Various UI Fixes, Referral email and Sidekiq

## Pull Request Name

#### Features

- Do not show static "No statements available" content
- Disable Click to verify button for broken uphold wallets
- Eyeshade::Balances, Eyeshade::Types, and Eyeshade::Client

#### How To Test

- Login and view dashboard, "No Statements available" no longer shows up
- Login as a user with a broken uphold wallet. DM me in slack if you need an account, I've got one I tested with

#### Pull Request Checklist
